src/main/java/com/aws/greengrass/shadowmanager/ipc/UpdateThingShadowRequestHandler.java

```diff
@@ -129,6 +132,10 @@ public UpdateThingShadowHandlerResponse handleRequest(UpdateThingShadowRequest r
                             .orElseThrow(() ->
                                     new InvalidRequestParametersException(ErrorMessage
                                             .createInvalidPayloadJsonMessage("")));
+                    if (updateDocumentRequest.has(SHADOW_DOCUMENT_STATE)
```

Comment:
I believe this would this be a breaking change for customers right? We don't process the `delta` node anyways. Why not just emit sending it to the cloud when we sync?

Reply:
it shouldn't break anyone. Anyone who is setting the delta today is already broken.

Reply:
The local update shouldn't be broken though right? This will break the local shadow update as well. It will only break the customers who are syncing that shadow with IoT Core.

Reply:
It isn't valid to set the delta. This isn't allowed by iot core, and it isn't in our documentation either. We say that you can set desired and reported, that's all.

Reply:
Mmm even if we don't say it on the documentation doesn't mean someone might not be passing a delta for local updates. Out of extreme caution maybe then we could just omit sending it sending it to Iot Core or if we are going to do this maybe remove the delta when receiving the update and log a warning.